src/components/NetworkSelector/NetworkSelector.js

```diff
@@ -48,15 +47,22 @@ export default function NetworkSelector(props) {
   }
 
   const onSelect = async (token) => {
+    const prevLabel = selectedLabel;
     setIsModalVisible(false);
     props.showModal(false);
     let network;
     try {
       network = await props.onSelect(token);
       setSelectedLabel(network);
+      const traits = {
+        previousNetwork: prevLabel,
+        currentNetwork: network,
+      };
+      trackAction("Network changed", traits);
```

Comment:
could just collapse this instead of setting the variable

Reply:
updated